370-auth-ui/oidc.md

```diff
@@ -290,6 +290,10 @@ The values will be `["sub", "iss", "aud", "exp", "iat"]`.
 
 The values will be `["plain", "S256"]`
 
+### revocation_endpoint
+
+No difference from the spec. See [Revocation Request](https://tools.ietf.org/html/rfc7009#section-2.1) for detail.
```

Comment:
It's better to document the endpoint. The authorization endpoint is `/oauth2/authorize` so the revocation endpoint could be `/oauth2/revoke`.

We should also document that the token endpoint is `/oauth2/token`.